demo/console.php

```diff
@@ -20,5 +20,8 @@
     $output->writeln('There was an error!');
     return 1;
 });
+$silly->command('sleep', function () {
```

Comment:
Why is this needed?

Reply:
To test what I implemented.

Those files are my "manual" test files ;)